db/db_creator.py

```diff
@@ -41,33 +41,60 @@ def __init__(
         )
         self.engine: Engine = create_engine(self.conn_str)
 
-        self.append_table_names: List[str] = append_table_names
-        self.append_table_names += DEFAULT_APPEND_TABLE_NAMES
-
     def get_table_names(self) -> List[str]:
+        '''
+        Gets table names using the SQLAlchemy Engine object.
+        '''
         logger.debug('get_table_names')
         return self.engine.table_names()
 
     def migrate(self) -> DBCreator:
+        '''
+        Updates database schema using yoyo-migrations and the migration files in the
+        migrations directory collocated with this file (db_creator.py).
+        '''
```

Comment:
You are returning the `self`. I never used/knew this kind of technique. Looking up quickly tell me this part of the Fluent interface approach you using with DBCreator.  I think to add a sentence in the doc-string top near the `DBCreator` will make the approach you're taken with DbCreator

Reply:
Okay, yes, I can add a clause or sentence to the top docstring stating that some methods can be used fluently.

Reply:
Okay, I updated that top docstring [here](https://github.com/tl-its-umich-edu/course-inventory/pull/129/commits/9ff82a7f427fe1fd1396c82e0234f8d9f61fbe57). Let me know if that helps.